lib/presentation/widgets/list_characters.dart

```diff
@@ -140,4 +148,15 @@ class _ListCharactersState extends State<ListCharacters> {
       fit: BoxFit.fill,
     );
   }
+
+  String _getErrorString(Object? error, BuildContext context) {
```

Comment:
It make sence to exctract it to some utils and place in presentation layer

Reply:
add it in extension on BuildContext

Reply:
done